examples/LiftSim_baseline/A2C/train.py

```diff
@@ -0,0 +1,226 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import os
+import parl
+import queue
+import six
+import time
+import threading
+
+from actor import Actor
+from collections import defaultdict
+from env_wrapper import ObsProcessWrapper, ActionProcessWrapper
+from parl.utils import logger, get_gpu_count, tensorboard, machine_info
+from parl.utils.scheduler import PiecewiseScheduler
+from parl.utils.time_stat import TimeStat
+from parl.utils.window_stat import WindowStat
+from rlschool import LiftSim
+from lift_model import LiftModel
+from lift_agent import LiftAgent
+
+
+class Learner(object):
+    def __init__(self, config):
+        self.config = config
+
+        #=========== Create Agent ==========
+        env = LiftSim()
+        env = ActionProcessWrapper(env)
+        env = ObsProcessWrapper(env)
+
+        obs_dim = env.obs_dim
+        act_dim = env.act_dim
+        self.config['obs_dim'] = obs_dim
+
+        model = LiftModel(act_dim)
+        algorithm = parl.algorithms.A3C(
+            model, vf_loss_coeff=config['vf_loss_coeff'])
+        self.agent = LiftAgent(algorithm, config)
+
+        if machine_info.is_gpu_available():
+            assert get_gpu_count() == 1, 'Only support training in single GPU,\
+                    Please set environment variable: `export CUDA_VISIBLE_DEVICES=[GPU_ID_TO_USE]` .'
+
+        else:
+            cpu_num = os.environ.get('CPU_NUM')
+            assert cpu_num is not None and cpu_num == '1', 'Only support training in single CPU,\
```

Comment:
I'm not sure if the assertion on CPU_NUM is still necessary now, as we have set it in the parl.compile function.  Please have a check on it and remove it if possible. https://github.com/PaddlePaddle/PARL/blob/c848bda2fcef55438b774e77ffba47d71df14418/parl/core/fluid/plutils/compiler.py#L36

Reply:
It is unnecessary now, removed.